pyproject.toml

```diff
@@ -181,6 +181,9 @@ Metrics = [
 ]
 # TODO: Add custom commands
 
+[tool.setuptools.package-data]
+"dipy" = ["py.typed"]
```

Comment:
`py.typed` and `meson.build` are enough.

I do not think we need this 2 lines in `pyproject.toml` since we do not use Setuptools at all. So I recommend to remove those lines. Let me know if I am missing something.

Reply:
:+1: Removed in 0ad2235. I do not know ways to test it, so I trust what your intuition is here.